package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Should have a django_react feature flag

Reply:
Unfortunately we cannot add comments to `.json` files. I will add a special note in the readme.

Reply:
Hmmm, could be worth adding the comment anyway to make it clear, though that would mean the default install will not work unless the comment is removed. Thoughts?

Reply:
I think that if we can't trust folks to read the readme, then they're going to have other problems. It would also mean (I think) that they'd have to remove the comment before installing any other kind of packages. IDK - the idea of having a file which is intentionally malformatted in the repo doesn't sit right with me.